doc/lsst.cp.verify/using-cp-verify.rst

```diff
@@ -0,0 +1,81 @@
+
+The outputs of `IsrTask` are best thought of as "residual" images showing what remaining signal is still present.  As a different configuration is used for these processings than would be in a standard science analysis, these residual images use a different dataset type (``verifyBiasProc`` for bias verification) to distinguish them from the standard ``postISRCCD` products.
+
+These residual images are the main input to `CpVerifyBiasTask`, a subclass of `CpVerifyStatsTask`, along with the `IsrTask` task metadata (for historical reasons; this was used as the source of the empirical read noise estimate), and any ``isrStatistics`` results.  The parent class has a handful of other input connections, allowing catalog results to be verified, and uncorrected versions of the input exposure and catalog (which have not had the calibration applied).  These connections are discussed below as they apply to particular calibration products.
+
+For simple measurements, the `CpVerifyBiasConfig` defines a set of dictionary configuration entries that have string keys that correspond to a measurement name that will appear in the output results, with a value that maps in most cases maps to an `lsst.afw.math.statistics.Property` indicating what mathematical operation will be used to perform that measurement.  If these dictionary configurations have non-zero length, then they are used by the task to determine which methods will be used to do the measurement.  These configuration options include:
+
+* ``imageStatKeywords``:  these options define the tests that will be run on each amplifier segment in the image being processed.  All following ``ImageStatKeyword`` entries follow this same behavior.
+
+* ``unmaskedImageStatKeywords``:  as above, these options define the tests run ignoring all mask planes present.
+
+* ``uncorrectedImageStatKeywords``:  these options define the tests that will be run using the ``uncorrectedExp`` input connection.
+
+* ``crImageStatKeywords``:  these options define the tests that will be run after cosmic ray rejection has been applied.
+
+* ``normImageStatKeywords``:  these options define the tests that will be run after normalizing the image by the exposure time.
+
+* ``detectorStatKeywords``:  as all previous image analysis is done on a per-amplifier basis, these options define tests to be run on the full detector.  As these will nearly always depend on the calibration under consideration, there is no defined method in the base class, so subclasses will need to implement it.
+
+* ``metadataStatKeywords``:  these options define the tests that will be run on the input task_metadata input.  These tests need to be implemented by subclasses.
+
+* ``catalogStatKeywords``:  these options define the tests that will be run on the input catalog data.  As before, these tests need to be implemented by subclasses.
```

Comment:
What sort of catalogs are these?  And where are they input from?

Reply:
These are catalogs from `CharacterizeImageTask`.  The main case for this is in the brighter-fatter verification, where we expect that objects will have smaller sizes with BF correction than without.  Text updated to indicate that a prior step in the pipeline should generate these, and a pointer to `CharacterizeImageTask`.